src/index.ts

```diff
@@ -140,6 +140,7 @@ function checkDiskSpace(directoryPath: string, dependencies: Dependencies = {
 	 * @param directoryPath - The file/folder path from where we want to know disk space
 	 */
 	async function checkUnix(directoryPath: string): Promise<DiskSpace> {
+		console.log('path ', dependencies.pathNormalize(directoryPath), ' sep ', dependencies.pathSep)
```

Comment:
Console logging here, i guess only for debugging ?

Reply:
ah my bad. will remove